src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -396,6 +396,10 @@ private function evaluate( $directive_value, string $default_namespace, $context
 			}
 		}
 
+		if ( $current instanceof Closure ) {
+			$current = $current( $store );
+		}
```

Comment:
Context and state are treated separately everywhere else, I think it makes sense to use 2 arguments here:

```suggestion
		if ( $current instanceof Closure ) {
			$current = $current( $store['state'], $store['context'] );
		}
```